app/src/docs/_examples/tables/ColumnsConfig.example.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useCallback, useEffect, useState, useMemo } from 'react';
 import { DataTableState, DataColumnProps, useLazyDataSource, useUuiContext } from '@epam/uui-core';
-import { Text,  DataTable, Panel, IconButton, DataTableMods } from '@epam/promo';
+import { Text,  DataTable, Panel, IconButton } from '@epam/promo';
+import { DataTableMods } from '@epam/uui';
```

Comment:
This component don't have props and as a result a typings for this props. So you can remove this import and props parameter also.